### PR TITLE
defects fixed and add eslint:recommended

### PR DESCRIPTION
**Pull request by ChrisH404**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  'extends': 'google',
+  'extends': ["eslint:recommended", "google"],
```
Comment (beltekylevi) [inline .eslintrc.js]: single quotes in js files!

**Review by beltekylevi — CHANGES_REQUESTED** [review]
run the linter, it should throw some errors in the `.eslintrc.js`

also jest environment should be enabled, are you sure its working properly?

File: package.json [hunk 2]
```diff
@@ -29,16 +29,19 @@
     "react-dom": "^15.6.2"
   },
   "devDependencies": {
-    "babel-eslint": "^8.0.1",
     "babel-core": "^6.26.0",
+    "babel-eslint": "^8.0.1",
```
Comment (beltekylevi) [inline package.json]: you can uninstall this dependency since you aren't using it

File: .eslintrc.js [hunk 3]
```diff
@@ -15,4 +18,46 @@ module.exports = {
...
   ],
+  'overrides': [
+    {
+      'files': ['src/client/**/*.js'],
+      'parserOptions': {
+        'sourceType': 'module',
+      },
+      'env': {
+        'browser': true,
+        'node': true,
```
Comment (beltekylevi) [inline .eslintrc.js]: do you really need the node environment enabled in the front end code?

Comment (ChrisH404) [reply .eslintrc.js]: we need to use 'require', it seems to be node grammar.

**Review by beltekylevi — APPROVED** [review]

**Pull request merged**